### ISSUE TITLE
No back icon shows in step 'Select Function App in Azure' on command plattee

### ISSUE THREAD

**Issue by v-xianya** [microsoft/vscode-azurefunctions#1946]
**OS**: Windows
**Build Version**: [20200228.9](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=15481&view=results)
**Regression:** Not a Regression

**Repro Steps:**
1. Click 'Deploy to Function App...' icon (Make sure there are more than one subscriptions).
2. Select one subscription.
3. Check whether there shows back icon in step 'Seletc Function App in Azure'.

**Expect:**
The back button shows well.

**Actual:**
No back button shows.
![image](https://user-images.githubusercontent.com/41351993/75681896-73dbed00-5ccf-11ea-8822-883aaf0c9ddc.png)
**More Info:**
This issue also reproduces for the  step after selecting function app.
![image](https://user-images.githubusercontent.com/41351993/75679006-c6b2a600-5cc9-11ea-92a2-3c1602d93fdd.png)

**Comment by EricJizbaMSFT** [comment]
Yeah this affects most command palette commands in all our extensions. We only use the wizard for things like "Create Function App", but could theoretically expand that to basically everything
